src/app/components/data-request/data-request.menus.ts

```diff
@@ -6,10 +6,12 @@ import { StrongRoute } from "@interfaces/strongRoute";
 export const dataRequestRoute = StrongRoute.newRoot().add(
   "data_request",
   (params: Params) => ({
-    userId: params?.userId,
-    projectId: params?.projectId,
+    // Handle params from workbench-client and baw-client
+    userId: params?.userId ?? params?.selected_user_id,
+    projectId: params?.projectId ?? params?.selected_project_id,
     regionId: params?.regionId,
-    siteId: params?.siteId,
+    siteId: params?.siteId ?? params?.selected_site_id,
+    timezoneName: params?.timezoneName ?? params?.selected_timezone_name,
```

Comment:
I appreciate that you added support for this, but this link will never be generated by the old-client, and the server will never need to redirect a user to this page with the old style QSPs, so redundant?